apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -453,3 +454,35 @@ API.v1.addRoute(
 		},
 	},
 );
+API.v1.addRoute(
```

Comment:
Instead of spreading all the endpoints created for this verification feature across multiple files, how about we group all of them together and put them in a single file?